Projects/UOContent/Regions/GuardedRegion.cs

```diff
@@ -286,19 +292,8 @@ public void CheckGuardCandidate(Mobile m)
 
             if (fakeCall != null)
             {
-                fakeCall.Say(
-                    Utility.RandomList(
-                        1007037,
-                        501603,
-                        1013037,
-                        1013038,
-                        1013039,
-                        1013041,
-                        1013042,
-                        1013043,
-                        1013052
-                    )
-                );
+                fakeCall.Say(Utility.Random(1013037, 16));
```

Comment:
- Removed `1007037` because it is from an older system of NPCs that you fight that would spam you with stupid messages that nobody cared about.
- Removed `501603` since it is a copy of another one we already have.
- Added some missing messages.